apps/src/blockly/customBlocks/googleBlockly/mutators/behaviorDefMutator.js

```diff
@@ -61,6 +61,10 @@ export const behaviorDefMutator = {
    * @this {Blockly.Block}
    */
   domToMutation: function (xmlElement) {
+    this.procedureName = xmlElement.nextElementSibling.getAttribute('id');
```

Comment:
It there documentation that explains why `nextElementSibling.getAttribute('id')` is the procedure name?